dataframely/_base_schema.py

```diff
@@ -137,6 +137,25 @@ def __new__(
                         f"which are not in the schema: {missing_list}."
                     )
 
+        # 3) Check that all members are non-pathological (i.e., user errors).
+        for attr_name, attr_value in namespace.items():
+            if attr_name.startswith("_"):
+                continue  # skip internal names
+
+            # Check for tuple of column (commonly caused by trailing comma)
+            if isinstance(attr_value, tuple):
```

Comment:
What about raising this only when the tuple has length 1? (to be on the conservative side)